### PR TITLE
Milestone 1: Testing add and remove functions

### PR DESCRIPTION
**Pull request by skabeo**
**In this pull request we;**

- Added Jest tests to ensure the add and remove functions are working as expected.
- And also the tests cover adding and removing items from an array, including edge cases like empty arrays and non-existent items. 
- This will help ensure the reliability of our code and prevent any potential issues in the future.

### CODE REVIEW

File: src/index.test.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    const addtask = addNewTask(task2);
+    expect(addtask).toBe(1);
+  });
+
+  test('should remove one task from the array.', () => {
+    const index = 1;
+    const removetask = deleteTask(index);
+    expect(removetask).toBe(1);
+  });
+});
```
Comment (AlphaNtihinduka) [inline src/index.test.js:16]: - [ ] [OPTIONAL] You did well to test adding and removing tasks using `toBe` assertion, which checks for strict equality.  However, depending on the implementation of the addNewTask and deleteTask functions, it may be more appropriate to use other assertions such as `toContain` or `not.toContain` to check if a task was added or removed from the list.

**Review by AlphaNtihinduka — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 
## Hi Team 🙋‍♂️🙋‍♂️, 

Your project is complete!  There is nothing else to say other than... Great job 👏 

approved-accepted

Congratulations! 🎉 
TO HIGHLIGHT:

- Your project is professional :heavy_check_mark: 
- Your repo is professional :heavy_check_mark: 
- Well-structured files :heavy_check_mark: 
- Professional README.md file 💯
- No linter errors :heavy_check_mark: 

Now you can merge this branch and move on :1st_place_medal: 


## Optional suggestions
- Kindly refer to the inline optional suggestions made. Ensure to apply them to have a professional project that suits recruiters' needs.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK WITH THE COMING ONES :crossed_fingers: :four_leaf_clover:

**Comment by MussieTeka** [comment]
We are merging because the PR has been approved in the code review process, and the Jest tests added will improve the code's reliability.
